packages/core/admin/server/routes/serve-admin-panel.js

```diff
@@ -29,7 +29,7 @@ const registerAdminPanelRoute = ({ strapi }) => {
   strapi.server.routes([
     {
       method: 'GET',
-      path: `${strapi.config.admin.path}/:path*`,
+      path: join(strapi.config.admin.path, '/:path*'),
```

Comment:
I think this won't work on windows machines, which are using `\` as separator and should use the [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) API instead. 

```
new URL("/:path*", strapi.config.admin.path).pathname
```

Reply:
This is a URL path not a file path, which is universal on all OSes

Reply:
Rhe URL API is what you'd want to build up URLs. This is not a file-path; therefore, it is just a coincidence `path.join` produces the same result.

I am not sure, but I think if you'd run node on a windows machine `path.join` probably uses backslash as a separator.